x-pack/plugins/fleet/server/services/output.ts

```diff
@@ -366,6 +413,7 @@ class OutputService {
 
   public async update(
     soClient: SavedObjectsClientContract,
+    esClient: ElasticsearchClient,
```

Comment:
All the changes where I added `esClient` as a second parameter depend from the call to `agentPolicyService.update` in this function, it needs to have the esClient and that impacted all the other functions that call it.